Tests/Routing/Loader/DirectoryRouteLoaderTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Tests\Routing\Loader;
+
+use FOS\RestBundle\Routing\Loader\DirectoryRouteLoader;
+use FOS\RestBundle\Routing\Loader\RestRouteProcessor;
+use Symfony\Component\Config\Loader\LoaderResolver;
+
+class DirectoryRouteLoaderTest extends LoaderTest
+{
+    public function testLoad()
+    {
+        $collection = $this->loadFromDirectory(__DIR__.'/../../Fixtures/Controller');
+
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('lock_user'));
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('ban_user'));
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('check_username_users'));
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('new_users'));
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('edit_user'));
+        $this->assertInstanceOf('Symfony\Component\Routing\Route', $collection->get('remove_user'));
```

Comment:
I am not really happy with this test. I think we should find a better way to ensure that the class does what it should.